### DISCUSSION TITLE
How can I hover on one element and css will change on the next element

### DISCUSSION THREAD

**Discussion by TechSinkV1** [tailwindlabs/tailwindcss#7054] [Help] [answered]
Hello guys, I want to hover on an icon and then style will apply to the next element which is after icon. Please tell me there is any way to do that.

**Comment by adamwathan** [comment, marked as answer]
Hey! You’re looking for this: https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-sibling-state

**Reply by TechSinkV1** [reply]
thanks
